docs/use-cases/portfolio-management/README.md

```diff
@@ -0,0 +1,44 @@
+---
+sidebar_navigation:
+  title: OpenProject Use-Cases: Portfolio Management and Custom Reporting
+  priority: 999
+description: Step-by-step instruction about portfolio management and custom reporting
+robots: index, follow
+keywords: help, documentation, use-cases, portfolio management
```

Comment:
Please don't use "help" and "documentation" as keywords for specific topic pages. (Only the root page should have them).